src/WinRT.Runtime/ObjectReference.cs

```diff
@@ -474,6 +474,31 @@ public static ObjectReference<T> Attach(ref IntPtr thisPtr)
             }
         }
 
+        public static ObjectReference<T> Attach(ref IntPtr thisPtr, Guid iid)
```

Comment:
Could we not make the `Attach` overload without iid just call this with `default(Guid)`?

Reply:
Possibly, but let's get this in to unblock. We can make more changes once this goes in.